plasmapy/diagnostics/charged_particle_radiography/tests/test_detector_stacks.py

```diff
@@ -0,0 +1,124 @@
+"""
+Tests for proton radiography functions
```

Comment:
Not for the first time, I am lamenting the lack of emoji related to charged particle radiography!  😿

Reply:
:)